packages/jmix-react-ui/src/ui/table/DataTable.tsx

```diff
@@ -133,7 +133,11 @@ export interface DataTableProps<TEntity> extends MainStoreInjected, MetadataInje
   columnDefinitions: Array<string | ColumnDefinition<TEntity>>
 }
 
-export interface ColumnDefinition<TEntity> {
+export interface ResizableColumn {
+  resizable?: boolean;
+}
+
+export interface ColumnDefinition<TEntity extends ResizableColumn> {
```

Comment:
`TEntity` contains entity attributes, so it should not extend `ResizableColumn`.